cleanlab_studio/cli/dataset/schema_types.py

```diff
@@ -70,25 +71,20 @@ class SchemaMetadata:
     id_column: str
     modality: Modality
     name: str
-    filepath_column: Optional[str]
 
     @staticmethod
-    def create(
-        id_column: str, modality: str, name: str, filepath_column: Optional[str] = None
-    ) -> "SchemaMetadata":
+    def create(id_column: str, modality: str, name: str) -> "SchemaMetadata":
```

Comment:
nit: do an `from __future__ import annotations` at the top of this file and remove the quotes around `SchemaMetadata`